selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/AssistantFindPanel.java

```diff
@@ -139,6 +139,18 @@ public void clickOnActionNodeWithText(String visibleText) {
     throw new RuntimeException(exceptionMessage);
   }
 
+  public void clickOnActionNodeWithTextEqualsTo(String visibleText) {
+    for (int i = 0; i < getActionNodesCount(); i++) {
+      if (getActionNodeText(i).equals(visibleText)) {
+        getActionNode(i).click();
+        return;
+      }
+    }
+    String exceptionMessage =
+        String.format("Item with expected visible text: \"%s\" has not been detected", visibleText);
```

Comment:
In this case error message  should be slightly different:
`"Item which has visible text equals to \"%s\" has not been detected"`